drivers/node/ssh/ssh.go

```diff
@@ -195,7 +195,7 @@ func (s *SSH) TestConnection(n node.Node, options node.ConnectionOpts) error {
 	if useSSH() {
 		cmd = "hostname"
 	} else {
-		cmd = "cat /etc/hostname"
+		cmd = "date"
```

Comment:
this is okay for now but we could just use the same hostname command for both. Once we are done with 2.2, you can send another PR to use `cmd = "hostname"` both both. (i.e remove the if condition)